### PR TITLE
Add new RGTS 2022 original track in `osu! originals`

### PR DESCRIPTION
**Pull request by Walavouchey**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) @Pisapou @Ganondorf-1 (dunno if mappool section should be translated)

### CODE REVIEW

**Comment by Pisapou** [comment]
except for `Wildcard` that I would probably translate into `Joker` , everything seems ok. I still prefer to wait for Ganon's approval to confirm you to change Wildcard translation

**Comment by Ganondorf-1** [comment]
I don't think mappool brackets/sections should be translated at all; we have never translated "tiebreaker" or anything like that because doing so would be confusing (and pointless since even French tournaments use the English terms)

**Review by Pisapou — APPROVED** [review]
So everything is fine coucerning french translation

**Review by TicClick — APPROVED** [review]

**Pull request merged**
